net.c

```diff
@@ -627,9 +630,9 @@ deliver_remote(struct qitem *it)
 	port = SMTP_PORT;
 
 	/* Smarthost support? */
-	if (config.smarthost != NULL) {
-		host = config.smarthost;
-		port = config.port;
+	if (is_configuration_setting_enabled(CONF_SMARTHOST) == true) {
```

Comment:
no boolean comparison to boolean, i.e. use `if (is_something_happening)`, not `if (is_something_happening == true)`

Reply:
Okay